Modix.Services/Starboard/StarboardSetup.cs

```diff
@@ -0,0 +1,15 @@
+﻿using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+using Modix.Services.Messages.Discord;
+
+namespace Modix.Services.Starboard
+{
+    public static class StarboardSetup
+    {
+        public static IServiceCollection AddStarboard(this IServiceCollection services)
+            => services
+                .AddScoped<StarboardService>()
```

Comment:
`.AddScoped<IStarboardService, StarboardService>()`

Reply:
I love you <3